lib/swaps/Swaps.ts

```diff
@@ -192,9 +207,21 @@ class Swaps extends EventEmitter {
     // TODO: check that we have route to taker.
     // TODO: extract data needed for proper timelock calculation and share with taker
 
+    let makerCltvDelta = 0;
+    const cltvDeltaFactor = this.lndLtcClient.getCltvDelta() / this.lndBtcClient.getCltvDelta();
```

Comment:
Are we risking a divide by zero error here? We might want to put in a check first.

Reply:
Never mind, this would put us back to where we were before which is checking every time which is unnecessary since it's a value that shouldn't change, I was just concerned about the default value being 0, but there's a check to prevent that case if the client is enabled. Maybe then we can leave this as is and just make the `cltvDelta` property readonly so it can't be possibly be changed 0.

Reply:
Added a comment to make it clear that it can't be zero